src/common/cursor.js

```diff
@@ -5,15 +5,12 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 
-import {Effects} from "reflex";
```

Comment:
Bah, I did multiple runs of this with slightly different settings, and
seemingly forgot to test after the last one. Will push an update.